.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+jobs:
+  python-3.4:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.4
+
+    steps: &build
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "requirements.txt" }}
+
+      - run:
+          name: virtual-env
+          command: |
+            if [ ! -d "$HOME/.coala-venv" ]; then python3 -m venv ~/.coala-venv; fi
+            echo "source ~/.coala-venv/bin/activate" >> $BASH_ENV
+
+      - run:
+          name: pre
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install 8
+            nvm alias default node
+            npm install -g csslint
+
+      - run:
+          name: dependencies
+          command: |
+            pip install -r test-requirements.txt
+            pip install -r requirements.txt
+            pip install -r docs-requirements.txt
```

Comment:
Should this be pip3 install?

Reply:
@rohanrk see .misc/deps.sh . In both cases pip install was used not pip3

Reply:
These can all go in one `pip` invocation, and definitely do not put test dependencies before the coala dependencies.  `pip` does the dependencies in the order that it sees them, and we care about the coala dependency versions more than the test dependency versions.